closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -536,22 +555,16 @@ private static void setProperties(Properties props) {
             String jdkPropsName = entry.getKey();
             String propsNewValue = entry.getValue();
 
-            String propsOldValue = props.getProperty(jdkPropsName);
-            if (isNullOrBlank(propsOldValue)) {
-                propsOldValue = "";
-            }
```

Comment:
Please explain why we don't want to combine `propsOldValue` and `propsNewValue` (below) as before.

Reply:
We didn't want the property to append to what was there before, but rather completely override it.